git-extra/git-extra.install

```diff
@@ -219,9 +219,14 @@ test -d "$TMPDIR" || test ! -d "$TMP" || {\
 	# Re-enable at least AES256-cbc and AES128-cbc, as they are still used
 	# e.g. in VSTS
 	test ! -f /etc/ssh/ssh_config ||
-	grep -q '^Ciphers .*aes256-cbc,aes192-cbc' /etc/ssh/ssh_config ||
-	sed -i -e '/^[# ]*Ciphers /{s/^# *//;s/$/,aes256-cbc,aes192-cbc/}' \
-		/etc/ssh/ssh_config
+	{
+		# Revert change by prior versions of git-extra.
+		sed -i -e 's/^Ciphers aes128-ctr,aes192-ctr,aes256-ctr,aes128-cbc,3des-cbc,aes256-cbc,aes192-cbc$/#   Ciphers aes128-ctr,aes192-ctr,aes256-ctr,aes128-cbc,3des-cbc/' \
+			/etc/ssh/ssh_config
```

Comment:
I tried very hard not to touch files unless needed, whereas this unconditional `sed` call would always be called.

Does this alternative snippet do the job, too?

```sh
! grep -q '^Ciphers .*cbc' ||
sed -i -e 's/^Ciphers .*cbc/# &/' /etc/ssh/ssh_config
```

?

Reply:
Actually, better test for `! grep -q ^Ciphers [a-z].*cbc`, to avoid catching the edit below.

Reply:
Ok.